DigitalLearningSolutions.Data.Migrations/Scripts/TD-1334-UpdatingTermsAndCondition.sql

```diff
@@ -0,0 +1,10 @@
+--TD-1334 update TermsAndConditions
```

Comment:
This is not how we create a migration script. Please refer to Auldrin's PR and consult with him to see how this class should be made by implementing both the Up and Down methods of 'Migration'

Reply:
Please remove this .sql file. It is not required.